sonic_installer/bootloader/bootloader.py

```diff
@@ -75,6 +75,10 @@ def supports_package_migration(self, image):
         """tells if the image supports package migration"""
         return True
 
+    def verify_image_sign(self, image_path):
```

Comment:
Outside of the scope of this fix but my main grief here is that there was already a method to check for secureboot images called `verify_secureboot_image` (see above).
I think your feature should use this method instead of adding a new one with the exact same purpose.
Also if you look closely most methods in the `Bootloader` class `raise NotImplementedError` to ensure child classes implement all the features (or explicitely skip some)`

Reply:
Sorry for your grief. I decided not to implement the method as verify_secureboot_image because of follow things:
1. From what I can see, the actual verify_secureboot_image is doing something completely different (checking that it has a certificate). Our method is doing image signature verification concerning secure upgrade
2. naming is not what we intend to do - this verification is regarding image signature, not secureboot (maybe in the general scope of it but not regarding the boot itself)
3. verify_secureboot_image is called with a force flag in main.py - we would like to avoid such thing, as it prevents the whole secure notion
Do you suggest in Bootloader the method will raise NotImplementedError and child classes will have empty implementation of it?